packs/equipment/arachnolute.json

```diff
@@ -35,6 +35,13 @@
         },
         "quantity": 1,
         "rules": [
+            {
+                "domain": "performance",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.PlayingItem",
+                "option": "playing",
```

Comment:
Wondering if we should consider being more specific like with `playing:x`. I have no idea who would be carrying around 2+ instruments but the one man band is gonna have all his bonuses activate at once :D

Using mergeable suboptions could solve that but also seems like a big effort for something that very few players would encounter since it would only affect people with multiple instruments equipped.

Alright, I think I've talked myself out of it now.